recipe/meta.yaml

```
@@ -9,9 +9,9 @@ source:
   sha256: 3f52243b7e300cb0fbfb7621d30693cf21392ec5ead3bb32a31d44f2d6bf656a
 
 build:
-  number: 2
-  script: PYORC_SKIP_ORC_BUILD=1 PYORC_LIBRARIES="orc" {{ PYTHON }} -m pip install . -vv
-  skip: true  # [win]
+  number: 3
+  script: PYORC_SKIP_ORC_BUILD=1 PYORC_LIBRARIES="orc" {{ PYTHON }} -m pip install . -vv  # [not win]
+  script: set PYORC_SKIP_ORC_BUILD=1&&set PYORC_LIBRARIES=orc&&{{ PYTHON }} -m pip install . -vv  # [win]
```

FWIW it is possible to use a list with `script` for multiline support (for [example]( https://github.com/conda-forge/waf-feedstock/blob/4d870b7a90a7b2390d1565f84b0f8f8f6f243367/recipe/meta.yaml#L14-L17 )):

```suggestion
  script:
    - export PYORC_SKIP_ORC_BUILD=1      # [unix]
    - set PYORC_SKIP_ORC_BUILD=1         # [win]
    - export PYORC_LIBRARIES="orc"       # [unix]
    - set PYORC_LIBRARIES="orc"          # [win]
    - {{ PYTHON }} -m pip install . -vv
```

---

Thanks. That actually makes it much more readable. I'm a bit lazy to fix this right now, but I'll keep this in mind next time when I work around that.

---

Ofc! Actually it can be simpler with `script_env`. Updated the comment above accordingly.

No worries. It's not pressing. Went ahead and filed an issue to track ( https://github.com/conda-forge/pyorc-feedstock/issues/38 )